### PR TITLE
[Frontend] As a user, I can logout

### LINKED ISSUE #13: [Frontend] As a user, I can logout
### Why?
An authenticated user can logout from the system with the logout button.

### Acceptance Criteria
- [x] Path `/user/sign_out`
- [x] User can make a request to sign out by clicking the logout button from the screen

### LINKED ISSUE #40: Fix: Deploy failed in staging
## Issue

When merging to the development branch, the deploy action will deploy the latest to the staging but it has an issue with some configuration then it can't  be deployed successfully 🤣

## Expected

Is have to be green when deploy to staging ❇️

## Steps to reproduce

Go to https://github.com/Lahphim/go-crawler-challenge/actions?query=workflow%3ADeploy
Screen Shot 2564-02-15 at 11 38 00

### PR DESCRIPTION
**Pull request by Lahphim**
Resolve https://github.com/Lahphim/go-crawler-challenge/issues/13
Resolve https://github.com/Lahphim/go-crawler-challenge/issues/40

## What happened 👀

In screen should show that the user has successfully signed in.
So the user can:
- See their avatar
- Navigate to home page
- Navigate to dashboard page
- Sign out

## Insight 📝

- [x] User can click sign out
- [x] User can sign out
- [x] Redirect to root page after signed out

Extra:

- Add logo with clickable to the root page
- Add navigation menu to `home` and `dashboard` page
- Remove alert message after 5 seconds

## Proof Of Work 📹

| Mobile Screen | Desktop Screen |
|---------------|-----------------|
| Screen Shot 2564-02-19 at 16 46 31 | Screen Shot 2564-02-19 at 16 46 13 |

| Before sign in | After sign in |
|---------------|-----------------|
| Screen Shot 2564-02-19 at 16 28 34 | Screen Shot 2564-02-19 at 16 29 03 |

Demo
Screen Recording 2564-02-19 at 16 50 25

### CODE REVIEW

**Review by olivierobert — COMMENTED** [review]
Early review: prefer avoiding single-word class name to avoid conflicts

> Class names must be composed of at least two words to make each element class name specific and more descriptive.
> [Compass](https://nimblehq.co/compass/development/code-conventions/css/#class-names)

File: views/shared/header.html [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+      <img width="48" height="48" class="brand-logo__icon" src="/assets/images/favicon.svg" alt="X42">
+      <span class="brand-logo__name">X42</span>
+    </a>
+  </div>
+
+  <div class="navbar-panel__navigation">
+    <div class="user-panel">
+      {{ if .CurrentUser }}
+        <img width="40" height="40" class="user-panel__avatar" title="{{ .CurrentUser.Email }}" src="https://www.gravatar.com/avatar/{{ .CurrentUser.Email | hashEmail }}?d=retro&f=y" alt="avatar">
+        <ul class="user-panel__action">
```
Comment (carryall) [inline views/shared/header.html]: @Lahphim IMO it's a bit weird when there's no user signed in and all the navigation links are under `user-panel` 😆 
Screen Shot 2564-02-19 at 17 22 51

Can we move all the links out?

File: assets/javascripts/components/alert/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+export const DEFAULT_SELECTOR = 'div.alert';
+
+const DEFAULT_OPTIONS = {
+    hideAfter: 5000,
```
Comment (carryall) [inline assets/javascripts/components/alert/index.js]: @Lahphim IMO 5 seconds is a bit too long, how about 3 seconds?

**Review by junan — APPROVED** [review]

File: assets/javascripts/components/alert/index.js (hunk 2 above)
Comment (Lahphim) [reply assets/javascripts/components/alert/index.js]: updated https://github.com/Lahphim/go-crawler-challenge/pull/53/commits/054f45d16e63e515a882de350bd71a946e42aae3

File: views/shared/header.html (hunk 1 above)
Comment (Lahphim) [reply views/shared/header.html]: updated https://github.com/Lahphim/go-crawler-challenge/pull/53/commits/054f45d16e63e515a882de350bd71a946e42aae3

**Review by carryall — APPROVED** [review]

**Pull request merged**